src/bundle/Resources/public/js/scripts/helpers/object.instances.js

```diff
@@ -0,0 +1,17 @@
+(function(global, doc, ibexa) {
+    const set = (domElement, instance) => {
+        domElement.ibexaInstance = instance;
+    }
+    const get = (domElement) => {
```

Comment:
Not sure whether we need to implement it right now, but... now it is not possible to check whether the given dom element has instance assigned without catching the error and checking its message.
Why not just return `undefined` when there is not instance assigned?
I think the guidance here may be `get` method from `Map` which does just that.
ref. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/get